virttest/ceph.py

```diff
@@ -165,8 +180,7 @@ def rbd_image_map(ceph_monitor, rbd_pool_name, rbd_image_name):
     :params rbd_pool_name: The name of rbd pool
     :params rbd_image_name: The name of rbd image
     """
-    cmd = "rbd map %s --pool %s -m %s" % (rbd_image_name, rbd_pool_name,
-                                          ceph_monitor)
+    cmd = "rbd map %s  --pool %s -m %s" % (rbd_image_name, rbd_pool_name, ceph_monitor)
```

Comment:
That's not different from before. And please check the code by pycodestyle. "line too long, > 79 characters"

Reply:
Fixed, thanks